server/src/test/java/org/elasticsearch/common/joda/JavaJodaTimeDuellingTests.java

```diff
@@ -33,9 +34,85 @@
 import java.util.Locale;
 
 import static org.hamcrest.Matchers.containsString;
+import static org.hamcrest.Matchers.equalTo;
 import static org.hamcrest.Matchers.is;
 
 public class JavaJodaTimeDuellingTests extends ESTestCase {
+    @Override
+    protected boolean enableWarningsCheck() {
+        // disable warning checks as deprecated patterns are used to compare Joda vs Java results (y - year and Z zone offset)
```

Comment:
I would update this note to say that these warnings as tested in `JodaWarningTests` already, so that's why we don't test them here.

Pretend I copy/pasted this onto every iteration of this comment, but I'm not going to because there's a lot of them.